doc/licenses/zxing-3.5.2/AUTHORS

```diff
@@ -73,14 +73,14 @@ Kevin Xue (NetDragon Websoft Inc., China)
 Lachezar Dobrev
 Loránt Gerencsér
 Luiz Silva
-Luka Finžgar
-Łukasz Stefaniak
+Luka Fin?gar
+?ukasz Stefaniak
```

Comment:
It looks like some characters got messed up and replaced with `?` here - is that correct?

Reply:
Ah, nope - that's from me copy/pasting the `AUTHORS` file through the meat grinder that is ISO 8859-1 (VNC clipboard). I'll correct this...

Reply:
Fixed via rebase.